### PR TITLE
fix https://github.com/Siemens-Healthineers/ETWAnalyzer/issues/60

### PR DESCRIPTION
**Pull request by Tejesh-SH**

### CODE REVIEW

File: ETWAnalyzer/Commands/DumpCommand.cs [hunk 1]
```diff
@@ -170,17 +170,18 @@ class DumpCommand : ArgParser
...
         "                         -SortBy Commit/SharedCommit Sort by Committed/Shared Committed (this is are memory mapped files, or page file allocated file mappings). " + Environment.NewLine + "" +
         "                                 WorkingSet/Diff    Sort by working set or committed memory difference" + Environment.NewLine +
         "                         -TopN dd nn                Select top dd processes. Optional nn skips the first nn lines of top list" + Environment.NewLine +
         "                         -TotalMemory               Show System wide commit and active memory metrics. Useful to check if machine was in a bad memory situation." + Environment.NewLine +
         "                         -MinMaxWorkingSetMiB xx-yy  Only include processes which had at least a working set xx-yy MiB (=1024*1024) at trace end. Numbers can have units like Bytes,KiB,MiB,GiB e.g. 500MiB." + Environment.NewLine +
+        "                         -MinMaxWorkingSetPrivateMiB xx-yy  Only include processes which had at least a working set private xx-yy MiB (=1024*1024) at trace end. Numbers can have units like Bytes,KiB,MiB,GiB e.g. 500MiB." + Environment.NewLine +
```
Comment (AloisKraus) [inline ETWAnalyzer/Commands/DumpCommand.cs:179]: The [MinMaxWorkingSetPrivateMiB  xx-yy] should show up also in the long list of options above.

Comment (AloisKraus) [reply ETWAnalyzer/Commands/DumpCommand.cs:179]: The same is true for -Details. It show up in the list of possible command line options for -Dump Memory

File: ETWAnalyzer/Commands/DumpCommand.cs [hunk 2]
```diff
@@ -1221,6 +1229,11 @@ public override void Parse()
                         KeyValuePair<decimal, decimal> minmaxsharedcommit = minmaxsharedcommitMBStr.GetMinMaxDecimal(MiBUnit);
                         MinMaxSharedCommitMiB = new MinMaxRange<decimal>(minmaxsharedcommit.Key / MiBUnit, minmaxsharedcommit.Value / MiBUnit);
                         break;
+                    case "-minmaxworkingsetprivatemib":
+                        string minmaxworkingsetprivateMBStr = GetNextNonArg("-minmaxworkingsetprivatemib");
```
Comment (AloisKraus) [inline ETWAnalyzer/Commands/DumpCommand.cs:1233]: minmaxworkingsetprivateMBStr  should be named minmaxworkingsetprivateMiBStr to not confuse the readers about the intended units. MiB are multiples of 1024 * 1024 and MB are multiples of 1000 * 1000.

File: ETWAnalyzer/EventDump/DumpMemory.cs [hunk 3]
```diff
@@ -362,23 +380,40 @@ private void Print(List<Match> matches)
...
                 }
 
                 if (IsSummary && printedFiles > 1)
                 {
-                    ColorConsole.WriteEmbeddedColorLine($"[cyan]Memory Total per File:[/cyan] [{GetTrendColor(totalDiff)}]TotalDiff: {totalDiff} [/{GetTrendColor(totalDiff)}] [{GetColorTotal(totalCommitedMemMiB)}] TotalCommitedMem: {totalCommitedMemMiB} MiB [/{GetColorTotal(totalCommitedMemMiB)}] [Darkyellow] Number of Involved Processes: {processCount} [/Darkyellow]");
+                    ColorConsole.WriteEmbeddedColorLine($"[cyan]Memory Total per File:[/cyan] [{GetTrendColor(totalDiff)}]" +
+                        $"TotalDiff: {totalDiff} [/{GetTrendColor(totalDiff)}] " +
+                        $"[{GetColorTotal(totalCommitedMemMiB)}] TotalCommited: {totalCommitedMemMiB} MiB [/{GetColorTotal(totalCommitedMemMiB)}] " +
+                        (IsFileTotalMode ? 
+                            $"[{GetColorTotal(totalWorkingsetPrivateMemMiB)}] TotalWorkingsetPrivate: {totalWorkingsetPrivateMemMiB} MiB [/{GetColorTotal(totalWorkingsetPrivateMemMiB)}]" :
```
Comment (AloisKraus) [inline ETWAnalyzer/EventDump/DumpMemory.cs:414]: The totals should be width aligned. Like {totalDiff}  -> {totalDiff,6} , {totalCommitedMemMiB} -> {totalCommitedMemMiB,6} {totalWorkingsetPrivateMemMiB}  -> {totalWorkingsetPrivateMemMiB,6}

**Review by AloisKraus — CHANGES_REQUESTED** [review]

File: ETWAnalyzer/Commands/DumpCommand.cs (hunk 1 above)
Comment (Tejesh-SH) [reply ETWAnalyzer/Commands/DumpCommand.cs:179]: resolved

**Pull request merged**